docs/4.1. Behaviour - Node Start-Up.md

```diff
@@ -0,0 +1,24 @@
+# Behaviour: Node Start-Up
+
+_(c) AMWA 2020, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
+
+This document covers expected behaviour for interactions between a Node (the client) and the System API (the server) at start-up.
+
+## Normal Operation
+
+1. The Node performs a DNS-SD browse for services of type '\_nmos-system.\_tcp' as specified by the client interaction procedure in [Discovery: Operation](3.1.%20Discovery%20-%20Operation.md).
+2. The Node fetches the global configuration resource (`/global`).
+3. The Node should use the unique System ID and the version timestamp (`id` and `version`) to determine whether the system environment has changed since its last start-up, for example, by comparing against information held in non-volatile storage.
+4. After successfully fetching the global configuration resource, a Node is not required to periodically repeat this operation until its next restart.
```

Comment:
By way of explanation, point 3 and the sentence following this list are intended to give TR-1001-1 hooks to hang its more stringent requirements on, without mandating those in this spec, which may be used by additional NMOS Application Profiles. Point 4 is included since some applications MAY choose to recommend periodic refresh, though TR-1001-1 currently says 'SHOULD NOT'.

Reply:
Is it perhaps worth calling this out explicitly? A sentence below the numbered list could indicate that external documents may add to or modify this list as required by their use cases.

Reply:
Good idea. Done.